src/browser/modules/Stream/SysInfoFrame/SysInfoFrame.tsx

```diff
@@ -107,26 +153,33 @@ export class SysInfoFrame extends Component<
         this.timer && clearInterval(this.timer)
       }
     }
+
+    if (prevProps.useDb !== this.props.useDb) {
+      this.getSysInfo()
+    }
   }
 
   getSysInfo(): void {
```

Comment:
let's make this an arrow function / class property to get rid of the need to bind it...